src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj

```diff
@@ -57,7 +57,7 @@
       <DependentUpon>NuGetCommand.resx</DependentUpon>
     </Compile>
     <EmbeddedResource Update="NuGetResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
Is this needed? If not it might be better to just remove this csproj change from the PR.